src/index.ts

```diff
@@ -15,21 +15,27 @@ import {
     ParameterDeclaration,
 } from 'ts-simple-ast';
 
+import { argv } from 'yargs';
+
 const ast = new AST();
 const tab = '    ';
 const mockClassFilenameSuffix = '.mock.ts';
 
-const passedArguments: string[] = process.argv.slice(2);
-const defaultSourcePath = './src/**/*.ts';
 const rootPath = process.cwd().replace( /\\/g, '/' );
-const providedSourcePaths = passedArguments.map( args => rootPath + args.replace( /^(\.)/g, '' ));
 
-if (providedSourcePaths.length > 0) {
-    ast.addSourceFiles( ...providedSourcePaths );
+let providedSourcePaths;
+
+const srcDir = rootPath + ( argv.src ? argv.src.replace( /^(\.)/g, '' ) : '/src');
+const outDir = rootPath + ( argv.out ? argv.out.replace( /^(\.)/g, '' ) : '/test');
+const inputs = argv._;
+
+if ( inputs && inputs.length > 0 ) {
+    providedSourcePaths = inputs.map( args => rootPath + args.replace( /^(\.)/g, '' ));
 } else {
-    ast.addSourceFiles( defaultSourcePath );
+    providedSourcePaths = [ ( srcDir.lastIndexOf( '/**/*.ts' ) == -1 ) ? srcDir + '/**/*.ts' : ''];
```

Comment:
The `src` input should not have `/**/*.ts`. Remove the check because we don't want it to work accidentally.